src/domain/model/character.go

```diff
@@ -0,0 +1,19 @@
+package model
+
+type Character struct {
+	Id       int      `json:"id"`
+	Name     string   `json:"name"`
+	Status   string   `json:"status"`
+	Species  string   `json:"species"`
+	Type     string   `json:"type"`
+	Gender   string   `json:"gender"`
+	Origin   Nested   `json:"origin"`
+	Location Nested   `json:"location"`
+	Image    string   `json:"image"`
+	Episodes []string `json:"episode"`
+}
+
+type Nested struct {
+	Name string `json:"name"`
+	Url  string `json:"url"`
```

Comment:
By convention (most lints even show a warning) the `Url` fields should be renamed as `URL`